BookingsApi/BookingsApi.IntegrationTests/Database/Queries/GetJusticeUsersListQueryHandlerDatabaseTests.cs

```diff
@@ -32,7 +32,17 @@ public async Task Should_return_null_when_no_users_are_found()
         [Test]
         public async Task Should_return_users_list()
         {
-            var query = new GetJusticeUserListQuery();
+            var query = new GetJusticeUserListQuery(null);
+            var users = (await _handler.Handle(query));
+
+            users.Should().NotBeNull();
+        }
+        
+        [Test]
+        public async Task Should_return_users_list_when_term_is_passed()
+        {
+            var term = "term";
```

Comment:
It would be good to seed a known data set and then run the query with a term

Reply:
here we are testing only if passing a term is still returning a list

Reply:
I see. I thought the purpose of the test was to check the query returned a known an expected list of values